scripts/scripts.js

```diff
@@ -10,6 +10,31 @@
  * governing permissions and limitations under the License.
  */
 
+/**
+ * Create an element with ID, class, children, and attributes
+ * @param {Object} props to create the element
+ * @returns {HTMLElement} the element created
+ */
+export function createEl({
+  tag, className, id, html, attributes,
+}) {
+  const el = document.createElement(tag);
+  if (id) { el.id = id; }
+  if (className) { el.className = className; }
+
+  if (html && html instanceof HTMLElement) {
+    el.append(html);
+  } else if (html && typeof html === 'string') {
+    el.innerHTML = html;
```

Comment:
`innerHTML` is pretty bad in terms of performance; a better approach would be to use https://developer.mozilla.org/en-US/docs/Web/API/Range/createContextualFragment